### PR TITLE
Hw 1 mingfeng yang

### PR DESCRIPTION
**Pull request by yomeho**

### CODE REVIEW

File: Homeworks/HW1/Mingfeng_yang/game_of_life.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+def evolve(initial_state):
+
+    # find the size of the data
+    rowlen=len(initial_state)
```
Comment (pusukuri) [inline Homeworks/HW1/Mingfeng_yang/game_of_life.py:4]: rowlen --> row_len    similarly other variables plz

File: Homeworks/HW1/Mingfeng_yang/game_of_life.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    collen=len(initial_state[0])
+    outcome=[line[:] for line in initial_state]
```
Comment (pusukuri) [inline Homeworks/HW1/Mingfeng_yang/game_of_life.py:6]: would be good to have a better name, e.g., next_state or evolved_state?

**Review by pusukuri — APPROVED** [review]
Great job...

File: Homeworks/HW1/Mingfeng_yang/game_of_life.py (hunk 1 above)
Comment (yomeho) [reply Homeworks/HW1/Mingfeng_yang/game_of_life.py:4]: Thanks, I will modify my mode of naming.

**Pull request merged**